test/nnstreamer_filter_nntrainer/runTest.sh

```diff
@@ -28,7 +28,7 @@ testInit $1
 PATH_TO_PLUGIN="../../build"
 
 if [[ -d $PATH_TO_PLUGIN ]]; then
-    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer/tensor_filter"
+    ini_path="${PATH_TO_PLUGIN}/nnstreamer/tensor_filter"
```

Comment:
I guess the path should be `${PATH_TO_PLUGIN}/test/nnstreamer_filter_nntrainer/nnstreamer-test.ini`

btw, I am not understanding this code bit.
It is finding ini_path and grep `nntrainer.so` which is not written in nnstreamer-test.ini

Reply:
IMO this is trying to find libnnstreamer_filter_nntrainer.so which is `build/nnstreamer/tensor_filter` because that's where it was built.

Reply:
I see. Thanks.